apartment/api/views.py

```diff
@@ -44,3 +49,29 @@ def get(self, request, project_uuid=None):
         )
         serializer = serializer_class(project_data, many=many)
         return Response(serializer.data)
+
+
+class ProjectExportApplicantsAPIView(APIView):
+    permission_classes = [permissions.AllowAny]
+    http_method_names = ["get"]
+
+    def get(self, request, project_uuid):
+        try:
+            apartment_uuids = get_apartment_uuids(project_uuid)
+            project = get_projects(project_uuid)[0]
+        except ObjectDoesNotExist:
+            raise NotFound()
+        reservations = ApartmentReservation.objects.filter(
+            apartment_uuid__in=apartment_uuids
+        )
+        export_services = ApplicantExportService(reservations)
+        csv_file = export_services.get_csv_string()
+        file_name = format_lazy(
+            _("[Project {title}] Applicants information"),
+            title=project.title,
```

Comment:
Is there actually `title` in project data? 

Also, personally I tend to avoid spaces in filenames as it makes them more cumbersome to be dealt with on command line, but that is probably just an outdated old fart thing, so feel free to disagree 🙂